### ISSUE TITLE
Lowercase "sirupsen" imports

### ISSUE THREAD

**Issue by jzelinskie** [Azure/draft#578]
If I import Go from this project, I'm forced to name my imports `github.com/Sirupsen/logrus` or else receive a compiler error:
```
formats.go:11:2: case-insensitive import collision: "github.com/sirupsen/logrus" and "github.com/Sirupsen/logrus"
```

The logrus project has [made a note][note] that eventually everyone should be using all lowercase for their imports.

It's very likely that one of Draft's dependencies is reason why it has to use this capitalization.

Hopefully by tracking the issue, we can convince others to lowercase their imports.

[note]: https://github.com/sirupsen/logrus/blob/90150a8ed11b6ce285e77e8af2b0109559ce4777/README.md#case-sensitivity

**Comment by bacongobbler** [comment]
This can be closed once #573 is merged :)

**Comment by bacongobbler** [comment]
closed via #573
